docs/user_guide.md

```diff
@@ -2,6 +2,7 @@
 
 ## Documentation for Previous Versions of NCEPLIBS-bufr
 
+* [NCEPLIBS-bufr-12.0.0](https://noaa-emc.github.io/NCEPLIBS-bufr/previous_versions/v12.0.0/index.html)
```

Comment:
I assume this URL will exist after the 12.0.1 release?

Reply:
Yes, once I officially release it and install the documentation (which just got generated in the Developer CI build ;-)